frontend/src/scenes/settings/environment/SessionRecordingIngestionSettings.tsx

```diff
@@ -320,6 +322,32 @@ function UrlBlocklistOptions(): JSX.Element | null {
     )
 }
 
+function EventTriggerOptions(): JSX.Element | null {
+    const { eventTriggerConfig } = useValues(sessionReplayIngestionControlLogic)
+    const { updateEventTriggerConfig } = useActions(sessionReplayIngestionControlLogic)
+
+    return (
+        <div className="flex flex-col space-y-2 mt-4">
+            <div className="flex items-center gap-2 justify-between">
+                <LemonLabel className="text-base">Event emitted</LemonLabel>
+            </div>
+            <p>Session recording will be started when the user sends any of these events.</p>
+            <EventSelect
+                filterGroupTypes={[TaxonomicFilterGroupType.CustomEvents]}
+                onChange={(includedEvents) => {
+                    updateEventTriggerConfig(includedEvents)
+                }}
+                selectedEvents={eventTriggerConfig ?? []}
+                addElement={
+                    <LemonButton size="small" type="secondary" icon={<IconPlus />} sideIcon={null}>
+                        Add event
+                    </LemonButton>
+                }
+            />
+        </div>
+    )
+}
+
 export function SessionRecordingIngestionSettings(): JSX.Element | null {
```

Comment:
I think I want to come back to the layout here since some of these controls are "and"

e.g. sampling and url triggering or sampling and linked flag

some are standalone e.g. url deny list should apply no matter what
and some are "or" e.g. url trigger or event trigger

i think that's fine but want to make this clear in the UI at some point
not blocking here since it's not a new confusion